Sources/Testing/Issues/Issue+Recording.swift

```diff
@@ -107,6 +107,34 @@ extension Issue {
     let issue = Issue(kind: .errorCaught(error), comments: Array(comment), sourceContext: sourceContext)
     return issue.record()
```

Comment:
This existing overload can call into the new one, below, passing `.error` for the severity